.genx/configure.js

```diff
@@ -8,33 +16,39 @@ module.exports = async (data, utils) => {
   // populate additional data fields
   data.pkgName = data.appName.replace(/[\W_]/g, '').toLowerCase();
   data.rootElement = `${data.pkgName}-root`;
-  data.localGenId = data.appName.toUpperCase().replace("-", "_");
+  data.localGenId = data.appName.toUpperCase().replace('-', '_');
   data.applicationVersionWeb = data.applicationVersion.split('-').shift();
   data.versions = versions;
 
   registerPartials(utils);
 
-  data.routes = data.routes
-    .filter(validateRoute)
-    .map(formatRouteData);
+  data.routes = data.routes.filter(validateRoute).map(formatRouteData);
 
-  const FDC3EventHandlersEnabled  = data.routes.find(route => route.FDC3EventHandlersEnabled);
+  const FDC3EventHandlersEnabled = data.routes.find(
+    (route) => route.FDC3EventHandlersEnabled,
+  );
   const FDC3ListenersEnabled = data.ui?.fdc3?.channels?.length;
   data.FDC3 = {
     includeDependencies: !!(FDC3ListenersEnabled || FDC3EventHandlersEnabled),
-    channels: data.ui?.fdc3?.channels || []
+    channels: data.ui?.fdc3?.channels || [],
   };
+  excludeFrameworks(data.framework);
 
-  data.routes.forEach(route => {
-    generateRoute(route, utils);
-  });
-
-  data.csv
-    .map(entity => ({
-      name: entity.name.toUpperCase(),
-      fields: entity.fields.map( (field, index) => ({name: field.toUpperCase(), isLast: index === (entity.fields.length -1) }))
-    }))
-    .forEach(entity => {
-      generateEmptyCsv(entity, data.appName, utils);
+  if (data.framework === FRAMEWORK_WEB_COMPONENTS_ALIAS) {
+    data.routes.forEach((route) => {
+      generateRoute(route, utils);
     });
+
+    data.csv
```

Comment:
CSV generation can be moved outside the web component if - it'll work the same for Angular too

Reply:
that's right, I removed the condition